x-pack/plugins/fleet/public/applications/integrations/layouts/default.tsx

```diff
@@ -20,18 +24,60 @@ interface Props {
   children?: React.ReactNode;
 }
 
+const Illustration = styled(EuiImage)`
+  margin-bottom: -68px;
+  width: 80%;
+`;
+
+const HeroImage = memo(() => {
+  const { toAssets } = useLinks();
+  const { uiSettings } = useStartServices();
+  const IS_DARK_THEME = uiSettings.get('theme:darkMode');
```

Comment:
I am wondering what is the best way to get that settings, you can also get it like this `useTheme().darkMode`

Reply:
I copied this component from the previous `header.tsx` file which was unused. I will update this to use the hook approach.